src/routes/components/components.json

```
@@ -2905,5 +2905,14 @@
 		"addedOn": "2023-02-24",
 		"category": "Integration",
 		"stars": 0
+	},
+	{
+		"title": "html-svelte-parser",
+		"url": "https://github.com/PatrickG/html-svelte-parser",
+		"description": "HTML to Svelte parser that works on both the server and the client.",
+		"npm": "html-svelte-parser",
+		"addedOn": "2023-02-26",
+		"tags": ["components and libraries", "layout and structure", "rich text editor", "ssr", "templates", "tree", "typescript"],
```

The linter (prettier) don't pass,
You need to run `npm run fomat` to fix it

Here what the `tags` node should look like:
```suggestion
		"tags": [
			"components and libraries",
			"layout and structure",
			"rich text editor",
			"ssr",
			"templates",
			"tree",
			"typescript"
		],
```